hydromt_wflow/version.py

```diff
@@ -0,0 +1,9 @@
+"""HydroMT-Wflow version."""
+
+main = 1
+major = 0
+minor = 0
+build = 0
```

Comment:
please use the actual semver names which is `major.minor.patch` if you want to do it like this